bridges/otelzap/core_test.go

```diff
@@ -27,12 +27,17 @@ func TestCore(t *testing.T) {
 	zc := NewCore(loggerName, WithLoggerProvider(rec))
 	logger := zap.New(zc)
 
-	logger.Info(testMessage)
+	logger.Info(testMessage, zap.String("key", "testValue"))
```

Comment:
Can you extract `"key"` to `testKey` variable and `"testValue"` for `testValue` variable and reuse it also in assertions?